scripts/release/publish-commands/publish-to-npm.js

```diff
@@ -8,10 +8,21 @@ const {join} = require('path');
 const semver = require('semver');
 const {execRead, execUnlessDry, logPromise} = require('../utils');
 
-const push = async ({cwd, dry, packages, version}) => {
+const push = async ({cwd, dry, packages, version, tag}) => {
   const errors = [];
-  const isPrerelease = semver.prerelease(version);
-  const tag = isPrerelease ? 'next' : 'latest';
+  let isPrerelease;
+  if (tag === undefined) {
+    // No tag was provided. Check the version.
+    isPrerelease = semver.prerelease(version);
+    if (isPrerelease) {
+      tag = 'next';
+    } else {
+      tag = 'latest';
+    }
+  } else {
+    // Any tag besides `latest` is a prerelease
+    isPrerelease = tag === 'latest';
```

Comment:
Isn't this the exact opposite of what you want? (Don't you want !==)

Reply:
Ha yes I hit this exact bug when I did the release and neglected to include the change in the commit :D